Source/Core/AudioCommon/CubebStream.h

```diff
@@ -0,0 +1,32 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <cstddef>
+#include <memory>
+
+#include "AudioCommon/SoundStream.h"
+
+#include <cubeb/cubeb.h>
+
+class CubebStream final : public SoundStream
+{
+public:
+  bool Start() override;
+  void Stop() override;
+  void SetVolume(int) override;
+
+private:
+  bool m_stereo{};
```

Comment:
This can be `bool m_stereo = false;` which is arguably more explicit and clear about what you mean.